scripts/reptiles.js

```diff
@@ -17,14 +17,94 @@ const h1Element = document.createElement('h1');
 heroDiv.appendChild(h1Element);
 
 const paragraphElement = document.createElement('p');
+paragraphElement.classList.add('text-paragraph');
 heroDiv.appendChild(paragraphElement);
 
 const btnElement = document.createElement('button');
 btnElement.classList.add('hero-btn');
 heroDiv.appendChild(btnElement);
 
-document.querySelector('h1').textContent = 'Welcome to the Reptile Page'
-document.querySelector('p').textContent = 'Ave Maria Gratia plena Mariagratia plena Maria gratia plena. Ave ave dominus, dominus tecum. Benedicta tu in mulieribus. Et benedictus. Et benedictus fructus ventris Ventris tuae, Jesus Ave Maria'
+
+
+
+document.querySelector('h1').textContent = 'Welcome to the Reptile Page';
+document.querySelector('.text-paragraph').textContent = 'Ave Maria Gratia plena Mariagratia plena Maria gratia plena. Ave ave dominus, dominus tecum. Benedicta tu in mulieribus. Et benedictus. Et benedictus fructus ventris Ventris tuae, Jesus Ave Maria';
 
 const btnText = document.createTextNode('Click!');
 btnElement.appendChild(btnText);
+
+//Read more button
+
+document.querySelector('.hb-read-more-text').textContent = 'The Hawksbill Sea Turtle gets its common name from the distinctive mouth, which resembles a birds beak. The shell, or carapace, is covered in large overlapping scutes, which are a distinctive brown/green/amber background with lighter brown streaks throughout. At the edge of the carapace they overlap in such a way as to form a serrated edge. The turtle has an elongated head and flippers which have two visible claws on the end. The flippers and head are covered in large green, brown or yellow scales. The average adult female weighs 50 kg and their carapace (shell) is approximately 80 cm in length.'
```

Comment:
The read more function looks good to me!

please note that the short summary contains the first 200 characters of the description (or a similar solution)​ 
Read more shows full text and  show all the properties of an animal.